README.rst

```diff
@@ -333,10 +333,24 @@ And then in the shell::
 What is the difference between `dockcross` and `dockbuild` ?
 ------------------------------------------------------------
 
-The key difference is that `dockbuild <https://github.com/dockbuild/dockbuild#readme>`_
-images use the same method to conveniently isolate the build environment as
-`dockcross <https://github.com/dockcross/dockcross#readme>`_ but they do **NOT** provide
-a toolchain file.
+The key difference is that `dockbuild
+<https://github.com/dockbuild/dockbuild#readme>`_ images do **NOT** provide
+a `toolchain file
+<https://cmake.org/cmake/help/latest/manual/cmake-toolchains.7.html`_
+but they use the same method
+to conveniently isolate the build environment as `dockcross
+<https://github.com/dockcross/dockcross#readme>`_.
+
+`dockbuild` is used to build binaries for Linux x86_64 / amd64 that will work
```

Comment:
Thank you for adding these paragraphs, the difference is crystal clear for me now.

Reply:
Thanks for the feedback, we will then integrate this PR and also add a similar text in dockbuild project.